samples/Tests/ColumnSet.Input.Text.VerticalStretch.json

```diff
@@ -0,0 +1,48 @@
+{
+	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+	"type": "AdaptiveCard",
+	"height": "stretch",
+	"version": "1.1",
```

Comment:
>1.1" [](start = 13, length = 4)

If we use v1.1 here this card won't render anymore because our renderer still thinks it's v1.0. We need to either keep our cards v1.0 for now, or update our renderer to claim version 1.1 (in AdaptiveCardStaticsImpl::FromJsonString)